db/kv_dense_channel_storage_test.go

```diff
@@ -573,12 +651,12 @@ func TestCalculateChangedPartitions(t *testing.T) {
 			assertTrue(t, partition == 0 || partition == 6 || partition == 12, "Unexpected changed partition")
 		}
 	}
-	assert.Equals(t, changedPartitions[0].GetSequenceRange(0).since, uint64(0))
-	assert.Equals(t, changedPartitions[6].GetSequenceRange(100).since, uint64(0))
-	assert.Equals(t, changedPartitions[12].GetSequenceRange(200).since, uint64(0))
-	assert.Equals(t, changedPartitions[0].GetSequenceRange(0).to, uint64(5))
-	assert.Equals(t, changedPartitions[6].GetSequenceRange(100).to, uint64(10))
-	assert.Equals(t, changedPartitions[12].GetSequenceRange(200).to, uint64(15))
+	assert.Equals(t, changedPartitions[0].GetSequenceRange(0).Since, uint64(0))
```

Comment:
This was breaking whenever max block size was changed, as the #overflow changed.

This could be re-enabled with the fix for 53, since it changes that const into a var that could be set by this unit test.